### PR TITLE
Pix fees

### PR DESCRIPTION
**Pull request by julianasobreira**
## Contexto
Incluir taxas de pix no empty state.

## Issues linkadas
- [x] [CRED-350](https://mundipagg.atlassian.net/browse/CRED-350)

### Preview:
image

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6761b57dcbc0c1d6c2208d471acc1699c605c89c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6761b57dcbc0c1d6c2208d471acc1699c605c89c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6761b57dcbc0c1d6c2208d471acc1699c605c89c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6761b57dcbc0c1d6c2208d471acc1699c605c89c
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/components/FeesDetails/FeeTitleAndValues.js [hunk 1]
```diff
@@ -14,23 +14,26 @@ const FeeTitleAndValues = ({ t, title, values }) => (
...
-            <p>{t(translationPath)}</p>
-            <p>
-              <strong>
-                {formattedValue} {t(valueSuffixPath)}
-              </strong>
-            </p>
-          </div>
-        )
-      })}
+                    return `${formattedValue} ${valueSuffix}`
```
Comment (zignd) [inline packages/pilot/src/components/FeesDetails/FeeTitleAndValues.js:29]: Tá dahora! Só encontrei um ponto que me deixou com dúvida, acho que esse espaçamento aqui no meio está gerando um caso onde na renderização ficam dois espaços do lado esquerdo do "+", mas não sei dizer se fica visível isso para o usuário quando a tela é renderizada. Veja aqui nesse trecho do snapshot.
image

Comment (julianasobreira) [reply packages/pilot/src/components/FeesDetails/FeeTitleAndValues.js:29]: Não tinha reparado, vou ajustar!

**Review by zignd — APPROVED** [review]

**Review by alexandrejuk — APPROVED** [review]

**Pull request merged**